### DISCUSSION TITLE
Calendar version update

### DISCUSSION THREAD

**Discussion by prachit1105** [kvyatkovskys/KVKCalendar#146] [Q&A] [answered]
Hi @kvyatkovskys , I am really appreciated you have consider my suggestion and made changes accordingly. but when I have updated my version to 0.4.5" negative sizes are not supported in the flow layout" Issue occurs.So I unable to update my project so waiting for your next version. Your files are not accessible as well directly so It is very difficult to solve issues.
Please give me suggestion or when will I expect your updated branch.Please do update ASAP.I am dependent on your library.I think your library is best just need some bug fixes.

**Comment by kvyatkovskys** [comment, marked as answer]
The version 0.4.6 is released 🙂
